Formula/gaudi.rb

```diff
@@ -0,0 +1,62 @@
+class Gaudi < Formula
+  desc "Provides necessary interfaces and services for building HEP exp frameworks"
+  homepage "https://gaudi.web.cern.ch/gaudi/"
+  url "https://gitlab.cern.ch/gaudi/Gaudi/-/archive/v35r0/Gaudi-v35r0.tar.gz"
+  sha256 "c01b822f9592a7bf875b9997cbeb3c94dea97cb13d523c12649dbbf5d69b5fa6"
+  license "Apache-2.0"
+  head "ssh://git@gitlab.cern.ch:7999/gaudi/Gaudi.git"
```

Comment:
Won't this need ssh authentication?

Reply:
It's a public repository. What would be the best ?

Reply:
Does something like `git://gitlab.cern.ch/gaudi/Gaudi.git` not work?

Reply:
Ok, this works: `https://gitlab.cern.ch/gaudi/Gaudi.git`

Reply:
Great, thanks. I will make the change